.github/workflow/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
```

Comment:
There's both a **workflow**/ci.yml (presumably ignored) and **workflows**/ci.yml

Reply:
Yes, thank you -fixed